.pending/improvements/sdk/4573-adds-height-in-

```diff
@@ -0,0 +1 @@
+#4573 adds height in response for validator endpoints
```

Comment:
can we make this available for all the endpoints?

Reply:
This also contradicts the PR title as it only updates another new endpoint. Should we update all querier endpoints?